src/lib/containers/drug_upload_tool/DrugUploadTool.tsx

```diff
@@ -78,16 +79,20 @@ class DrugUploadTool extends React.Component<
   getFileEntityData = async (
     token: string,
     entityId: string,
+    {doDereference = false} = {}
```

Comment:
I believe this signature could be simplified by making the doReference boolean its own argument instead of it being a key in a dictionary.

Reply:
Yes,  it's bad code agree. I will create  another fn for the dereferencing case